.storybook/recipes/PageShell/PageShell.module.css

```diff
@@ -25,6 +25,7 @@
 
   &:focus {
     transform: translateY(0%);
+    @mixin focusInverted;
```

Comment:
Good catch!

Do we know why it doesn't have a focus outline to begin with? Are we suppressing native outlines or something, and need to add this mixin where we want them?

Reply:
Actually, looking at it again, it does have a focus outline. The issue is that the default focus outline is the same color as the nav. So this is a case where what we really need is the dual focus outline because we don't know for sure what color background the link will appear in front of. I happened to be testing it at a window width where the mobile version of the nav is present, and it was presumably built while testing with a wider window so the nav was on the left.

The skip link in LP has an interesting solution: the link has a background color and the focus outline is offset -2px to place the outline inside the background color, so it will always be purple on white.

<img width="543" alt="learning platform skip link with dev tools open displaying the styles" src="https://user-images.githubusercontent.com/7761701/191304546-0118435d-7957-4cec-900c-9b1faa978d33.png">

Reply:
Also apparently Along doesn't have a skip link at all.. 🤔

Reply:
Anyway I'll close this PR then because:
- it doesn't solve the issue; just transforms it
- the thing I was fixing is just a recipe anyway, not a component
- the skip link in LP is fine and won't be broken when the nav starts moving to the top of the page 
- Along doesn't have a skip link to fix